.dockerignore

```diff
@@ -3,4 +3,3 @@ client/dist/
 node_modules/
 .tmp/
 .venv/
-.git/
```

Comment:
This was the easiest way I found to avoid `unknown-branch` on Percy. Does it impact elsewhere? :thinking:

Reply:
Yeah, it will unnecessarily increase the size of the Docker image. But I believe Percy can use env vars instead of using git directly: https://docs.percy.io/docs/environment-variables (not sure if this applies to the client we use though).

Reply:
Those env vars were actually my second option haha. I'll handle it with them.